### PR TITLE
feat: add `cargo-binstall` method

### PR DESCRIPTION
**Pull request by jaeheonji**
This PR resolve #1067 

This method will be applied in the next release.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -59,5 +59,10 @@ assets = [
   ["assets/completions/_zellij", "usr/share/zsh/vendor-completions/_zellij", "644"],
 ]
 
+[package.metadata.binstall]
+pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.{ archive-format }"
```
Comment (ken-matsui) [inline Cargo.toml:63]: According to [this comment](https://github.com/ryankurte/cargo-binstall/issues/38#issue-823602518), I would say the `archive-format` should be `tar.gz` :)

```suggestion
pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.tar.gz"
```

**Review by ken-matsui — COMMENTED** [review]
I just happened to be looking at your PR and found a possible problem, so please let me leave a comment, although I am not a part of a member of Zellij.

**Comment by jaeheonji** [comment]
@ken-matsui Thank you! members don't matter. please feel free to review :)

**Comment by NewtonChutney** [comment]
Huh, this wasn't ever added in docs!

**Comment by NewtonChutney** [comment]
Raised a pr for docs: https://github.com/zellij-org/zellij-org.github.io/pull/271

**Pull request merged**